File: Cpp/Motherboard.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class Motherboard : public Komponen {
private:
    string chipset;
public:
    Motherboard() {}

    Motherboard(string chipset, string merk, string nama) 
        : Komponen(merk, nama) {
        this->chipset = chipset;
    }

    void setChipset(string chipset) {
        this->chipset = chipset;
    }

    string getChipset() const { 
        return this->chipset;
    }
    

    ~Motherboard() {}
};

File: Python/Motherboard.py
from Komponen import Komponen

class Motherboard(Komponen):
    def __init__(self, merk, nama, chipset):
        super().__init__(merk, nama)
        self.__chipset = chipset

    # Getter
    def get_chipset(self):
        return self.__chipset

    # Setter
    def set_chipset(self, chipset):
        self.__chipset = chipset
